addons/binding/org.openhab.binding.sonos/src/main/java/org/openhab/binding/sonos/internal/SonosXMLParser.java

```diff
@@ -44,7 +44,7 @@
 					"<dc:title>{2}</dc:title>" +
 					"<upnp:class>{3}</upnp:class>" +
 					"<desc id=\"cdudn\" nameSpace=\"urn:schemas-rinconnetworks-com:metadata-1-0/\">" +
-					"RINCON_AssociatedZPUDN</desc>" +
+					"{4}</desc>" +
```

Comment:
:-) If Spotify works, then I will be a happy man. 

Initially, in the haydays, I did not have the time to figure out it had to be done.